### PR TITLE
Add onboarding to Openshift documentation

### PR DESCRIPTION
**Pull request by christophe-f**
## Description

Add section to onboard to Openshift

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

_Put an `x` in the boxes that apply_

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary

### CODE REVIEW

File: docs/application-migration/learning-path-for-developers.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+   3. Free Self-Paced Training [Running Containers with Red Hat Technical Overview](https://www.redhat.com/en/services/training/rh065-running-containers-red-hat-technical-overview) (1 hour)
+3. Learning Path on Kubernetes/OpenShift :
+   1. OpenShift Fundamentals:
+      1. Free Self-Paced Training: [Containers, Kubernetes and Red Hat OpenShift Technical Overview](https://www.redhat.com/en/services/training/do080-deploying-containerized-applications-technical-overview) (1 hour)
+      2. Free Self-Paced Training [Getting started with OpenShift](https://developers.redhat.com/learn/openshift/develop-on-openshift) (Optional)
+      3. [OpenShift Official Documentation](https://docs.openshift.com/container-platform/4.10/web_console/web-console-overview.html)
+      4. [Kubernetes Official Documentation](https://kubernetes.io/docs/concepts/overview/)
+4. Learning Path on CI/CD
+   1. What’s CI/CD
+      1. [https://www.redhat.com/en/topics/devops/what-cicd-pipeline](https://www.redhat.com/en/topics/devops/what-cicd-pipeline)
```
Comment (schultzp2020) [inline docs/application-migration/learning-path-for-developers.md:18]: When referring to link you should either substitute it for text or use `<>`.

```suggestion
      1. [What is a CI/CD pipeline?](https://www.redhat.com/en/topics/devops/what-cicd-pipeline)
```

or 

```suggestion
      1. <https://www.redhat.com/en/topics/devops/what-cicd-pipeline>
```

File: docs/application-migration/learning-path-for-developers.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      2. [https://www.redhat.com/en/topics/devops/what-is-ci-cd](https://www.redhat.com/en/topics/devops/what-is-ci-cd)
```
Comment (schultzp2020) [inline docs/application-migration/learning-path-for-developers.md:19]:
```suggestion
      2. [What is CI/CD?](https://www.redhat.com/en/topics/devops/what-is-ci-cd)
```

File: docs/application-migration/learning-path-for-developers.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+   2. [CI/CD with OpenShift](https://www.redhat.com/en/about/videos/learn-cloud-native-series-cicd-with-openshift)
+   3. DevOps Best Practices
+   4. Azure Pipelines training:
+      1. [https://learn.microsoft.com/en-us/azure/devops/pipelines/get-started/key-pipelines-concepts?view=azure-devops](https://learn.microsoft.com/en-us/azure/devops/pipelines/get-started/key-pipelines-concepts?view=azure-devops)
```
Comment (schultzp2020) [inline docs/application-migration/learning-path-for-developers.md:23]:
```suggestion
      1. [Key concepts for new Azure Pipelines users](https://learn.microsoft.com/en-us/azure/devops/pipelines/get-started/key-pipelines-concepts?view=azure-devops)
```

File: docs/onboard-openshift.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+The Developer Team processes to be ready to start their container journey.
+
+1. Developer Foundational Learning
+2. Application Assessment: Apply Twelve-Factor App for Containers.
+
+[A checklist for The Top Twelve-Factor App for Containers]()
+
+The application owner and the team will perform a high-Level application assessment to understand the application maturity level, risks, potential changes, and feasibility.
+
+- [ ] I. Codebase: One codebase tracked in revision control, many deploys
```
Comment (schultzp2020) [inline docs/onboard-openshift.md:22]: What do the roman numerals represent? An ordered list?

File: docs/onboard-openshift.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+## Summary
+
+### Developer Team Guide
+
+The Developer Team processes to be ready to start their container journey.
+
+1. Developer Foundational Learning
+2. Application Assessment: Apply Twelve-Factor App for Containers.
+
+[A checklist for The Top Twelve-Factor App for Containers]()
```
Comment (schultzp2020) [inline docs/onboard-openshift.md:18]: Missing link?

File: docs/onboard-openshift.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+This process creates all the required resources for the development team and applications based on their requirements.
+
+##### Platform Team Checklist
+
+Developers will require a new namespace, access to the namespace, and other requirements to have their applications functional.
+
+This is a recommended list to start creating your platform guide to successfully onboard teams and applications into OpenShift.
+
+- [ ] 1- Project Provisioning per Team
```
Comment (schultzp2020) [inline docs/onboard-openshift.md:47]: What do the numbers mean? 

image

**Review by schultzp2020 — CHANGES_REQUESTED** [review]

File: docs/onboard-openshift.md (hunk 6 above)
Comment (christophe-f) [reply docs/onboard-openshift.md:47]: This is to reflect the original document. The content will be updated by the editor.

File: docs/onboard-openshift.md (hunk 5 above)
Comment (christophe-f) [reply docs/onboard-openshift.md:18]: Yes the link is missing in the original document. For now it's a starting document with a skeleton and a few pages to help the editor to add more content.

**Pull request merged**